ewah/cleaner.py

```diff
@@ -116,7 +117,14 @@ def __init__(
 
         cleaning_steps = []
 
-        if exclude_columns:
+        if include_columns:
+            cleaning_steps.append(self._include_columns)
+            self.include_columns = include_columns
+
+        if include_columns and exclude_columns:
+            _msg = "Don't use include and exclude columns config at the same time!"
+            raise Exception(_msg)
+        elif exclude_columns:
```

Comment:
Style (non-blocking): I don't like `elif` unless absolutely necessary. You can do the same with a new `if` statement. --> Please use `if` instead of `elif`